src/schema/v2/artwork/artworkOrError.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  GraphQLFieldConfig,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import Artist from "schema/v2/artist"
+import { ResolverContext } from "types/graphql"
+import { ArtworkType, artworkResolver } from "schema/v2/artwork"
+import ArtworkLayers, { artworkLayers } from "./layers"
+
+const ArtworkErrorType = new GraphQLObjectType<any, ResolverContext>({
+  name: "ArtworkError",
+  description:
+    "An error type, potentially containing a partial artwork response",
+  fields: {
+    artwork: {
+      type: new GraphQLObjectType<any, ResolverContext>({
+        name: "PartialArtwork",
+        fields: {
```

Comment:
These fields are as an example - imagine the error page showing the artist/artworks (with artist follow control), as well as a grid of related artworks (via `layers`).

We can flush them out better as we decide on what to show.